scripts/gen_eval_files.lua

```diff
@@ -337,9 +337,30 @@ local function render_api_keyset_meta(_f, fun, write)
   end
 end
 
+--- @param x string
+--- @return string
+local function dedent(x)
+  local xs = split(x)
+  local leading_ws = xs[1]:match('^%s*') --[[@as string]]
+  local leading_ws_pat = '^' .. leading_ws
+
+  for i in ipairs(xs) do
+    local strip_pat = xs[i]:match(leading_ws_pat) and leading_ws_pat or '^%s*'
+    xs[i] = xs[i]:gsub(strip_pat, '')
+  end
+
+  return table.concat(xs, '\n')
```

Comment:
This function is too complicated. Each file should just have a hard-coded indent to be stripped.